hazelcast-integration/kubernetes/samples/springboot-k8s-hello-world/pom.xml

```diff
@@ -22,13 +22,13 @@
 
 		<docker.image.prefix>${project.artifactId}</docker.image.prefix>
 
-		<java.version>8</java.version>
+		<java.version>1.8</java.version>
 
 		<!-- Latest versions -->		
 		<dockerfile-maven-plugin.version>1.4.12</dockerfile-maven-plugin.version>
-		<hazelcast.version>3.12.2</hazelcast.version>
+		<hazelcast.version>3.12.6</hazelcast.version>
```

Comment:
I've had troubles running client for this code sample but it seems to work with 3.12.6.